From 91f92c088aeb8078b0778d6cebfc5f64cf90a5ff Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Sat, 10 Feb 2018 20:32:43 -0500
Subject: [PATCH 10/27] workaround FindGSL.cmake bug

Omit FindGSL.cmake lines that do not work on Debian systems.
gr-wavelet fails with cmake 3.4.1 giving
Could NOT find GSL (missing:  GSL_INCLUDE_DIRS GSL_LIBRARY_DIRS)
without this workaround. Earlier invocation by gr-fec succeeds,
so likely some problem with use of CMake cache by FindGSL.cmake.
---
 cmake/Modules/FindGSL.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/cmake/Modules/FindGSL.cmake
+++ b/cmake/Modules/FindGSL.cmake
@@ -146,5 +146,5 @@
 
 ENDIF(GSL_FOUND)
 
-INCLUDE(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(GSL DEFAULT_MSG GSL_LIBRARIES GSL_INCLUDE_DIRS GSL_LIBRARY_DIRS)
+#INCLUDE(FindPackageHandleStandardArgs)
+#FIND_PACKAGE_HANDLE_STANDARD_ARGS(GSL DEFAULT_MSG GSL_LIBRARIES GSL_INCLUDE_DIRS GSL_LIBRARY_DIRS)
